packages/docs-theme/src/components/navigator.tsx

```diff
@@ -85,7 +85,7 @@ export class Navigator extends React.PureComponent<INavigatorProps> {
     }
 
     private filterMatches: ItemListPredicate<INavigationSection> = (query, items) =>
-        filter(items, query, {
+        filter([...items], query, {
```

Comment:
could also cast as a writable array instead. Problem here is the upstream types for `filter` from `fuzzaldrin-plus` don't use `readonly` for this argument